Added Instructions to Readme to allow connections from 0.0.0.0

W1BTR opened the pull request:
By default, backrest suggests connecting to localhost:9898, however, most Linux systems that I work with are headless, and backrest is being set up from an ssh client.

This pull request adds the following notes to the readme:




If the machine is headless, you will need to run this command (or manually edit `/etc/systemd/system/backrest.service` to enter 0.0.0.0).
```
sudo sed -i 's|Environment="BACKREST_PORT=172\.0\.0\.1:9898"|Environment="BACKREST_PORT=0.0.0.0:9898"|' /etc/systemd/system/backrest.service
```
_Note that this will allow web-ui connections from any device on your network. Always enable user authentication and set a password immediately after enabling this_

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/garethgeorge/backrest?pullRequest=672) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/garethgeorge/backrest?pullRequest=672) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/garethgeorge/backrest?pullRequest=672) it.</sub>

W1BTR commented:
Just modified the instructions to use `systemctl edit`. Also added a command to automatically download the latest release for 64 bit as an example (as before I've had to download the latest release, then upload it to the server Im installing backrest on).

garethgeorge commented:
Hey -- sorry for the late followup, I went ahead and reorganized the README.md and added this note re: editing the systemd unit while I was at it. Can you take a look and check if it captures what you were hoping / feel free to sync and add anything I've missed?

W1BTR commented:
No apologies needed, thank you for taking the time to look at it. Those instructions look great to me. Thanks!

Taking the liberty of closing this for now, feel free to re-open if you have more thoughts.
